File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findNumbers = function(nums) {
  const isEvenDigits = num => {
    if (num > 999) {
      return num <= 9999 || num == 100000
    } else {
      return num > 9 && num <= 99
    }
  }
  let ans = 0
  nums.forEach(n => ans += isEvenDigits(n))
  return ans
};

File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.go
func isEvenDigits(num int) bool {
  if num > 999 {
    return num <= 9999 || num == 100000
  } else {
    return num > 9 && num <= 99
  }
}

func findNumbers(nums []int) int {
  ans := 0
  for _, v := range nums {
    if isEvenDigits(v) {
      ans ++
    }
  }
  return ans
}